engine-tests/src/test/java/org/terasology/math/MatrixUtilsTest.java

```diff
@@ -0,0 +1,157 @@
+package org.terasology.math;
```

Comment:
I heavily refactored the MatrixUtils. These tests verify that MOST of it is behaving just like the old implementation.
MOST because I'm pretty sure to have detected an issue in the old implementation. I'll point to that in the appropriate change.